lib/jsdom/living/validity/ValidityState.webidl

```diff
@@ -0,0 +1,15 @@
+[Exposed=Window]
+interface ValidityState {
+  readonly attribute boolean valueMissing;
+  readonly attribute boolean typeMismatch;
+  readonly attribute boolean patternMismatch;
+  readonly attribute boolean tooLong;
+  readonly attribute boolean tooShort;
+  readonly attribute boolean rangeUnderflow;
+  readonly attribute boolean rangeOverflow;
+  readonly attribute boolean stepMismatch;
+  readonly attribute boolean badInput;
+  readonly attribute boolean customError;
+
```

Comment:
We'd prefer to keep the IDL close to the way it looks in the spec to make updating it easier in the future. The IDL in the spec doesn't appear to include a blank line between the errors and the `valid` attribute here.